core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -201,11 +201,38 @@
         };
         $scope.hasCustomMarkup = function(item) {
           return typeof _.result(item, 'markup') !== 'undefined';
-        }
+        };
 
         $scope.getCustomMarkup = function(item) {
           return $sce.trustAsHtml(_.result(item, 'markup') || '');
-        }
+        };
+
+        $scope.getShortcut = function(itemShortcut) {
+         function replace(str) {
+            if (bkHelper.isMacOS) {
+              var mapObj = {
+                Cmd: "&#x2318;",
+                Alt: "&#x2325;",
+                Shift: "&#x21E7;",
+                Up: "&#x2191;",
+                Down: "&#x2193;"
+              };
+              var regexp = new RegExp(Object.keys(mapObj).join("|"),"gi");
+              str = str.replace(regexp, function(matched) {
+                return mapObj[matched];
+              });
+            }
+
+            return $sce.trustAsHtml(str);
```

Comment:
how can you bind unsafe string value with special symbols without marking it as trusted html? angular doesn't accept such binding. I don't understand why not to use it because it's [written in the documentation](https://github.com/angular/angular.js/blob/v1.5.0/src/ng/directive/ngBind.js#L150). Otherwise - include ngSanitize module, but does it make sense?